core/src/main/java/org/testcontainers/containers/SocatContainer.java

```diff
@@ -17,7 +17,7 @@
     private final Map<Integer, String> targets = new HashMap<>();
 
     public SocatContainer() {
-        this(TestcontainersConfiguration.getInstance().getSocatDockerImageName());
+        this(DockerImageName.parse("alpine/socat:latest"));
```

Comment:
not really related to this PR, but perhaps now is a good time to pin socat? :D 
Or re-publish it under `testcontainers`  as an automated build with `FROM alpine/socat:latest` to pin it?

Reply:
Good point!

Reply:
Done in 7b8cedf

Reply:
Nice! Although `SocatContainer` still refers `:latest` 😅

Reply:
Should be fixed now.